airsonic-main/src/main/java/org/airsonic/player/service/SonosService.java

```diff
@@ -116,26 +116,48 @@
     @Resource
     private WebServiceContext context;
 
-    public void setMusicServiceEnabled(boolean enabled, String baseUrl) {
+    /**
+     * Try to enable/disbale the Sonos link.
+     *
+     * @param enabled or not, is not the sonoslink table is clear
```

Comment:
`@param enabled or not, is not the sonoslink table is clear` → `@param Enable/disable the music service. If disabled, the sonoslink table is cleared`